next.config.js

```diff
@@ -6,8 +6,8 @@ const { PHASE_DEVELOPMENT_SERVER } = require("next/constants");
 module.exports = (phase) =>
   withImages(
     withMdxEnhanced({
-      // layoutPath: 'layouts',
-      // defaultLayout: true,
+      layoutPath: "layouts",
+      defaultLayout: true,
```

Comment:
Applies `layouts/index.tsx` by default if no layout is specified in the frontmatter